components/rmrk/Gallery/GalleryItemCarousel.vue

```diff
@@ -47,7 +47,19 @@ export default class GalleryItemCarousel extends Vue {
         },
       })
 
-      this.nfts = await formatNFT(data?.collectionEntityById?.nfts)
+      const nftsWithCollectionId = data?.collectionEntityById?.nfts.map(
+        (nft) => {
```

Comment:
you can do 

```js
(nft) => ({
  //whatever
})
```

Instead of 

```js
(nft) => {
  return {
    //whatever
  }
}
```

Reply:
Thantks that was the shorthand I was looking for